source/common/common/hash.cc

```diff
@@ -0,0 +1,68 @@
+#include "common/common/hash.h"
+
+#include "absl/strings/string_view.h"
+
+namespace {
+
+// used by MurmurHash::murmurHash2_64
```

Comment:
For clarity just make all of these functions private static functions in the MurmurHash class.